ui/src/utils/openUrl.ts

```diff
@@ -3,13 +3,8 @@
  *  Licensed under the MIT License. See License.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// tslint:disable-next-line:no-require-imports
-import opn = require("opn");
+import * as vscode from 'vscode';
 
 export async function openUrl(url: string): Promise<void> {
-    // Using this functionality is blocked by https://github.com/Microsoft/vscode/issues/25852:
-    // await vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(url));
-
-    // tslint:disable-next-line: no-unsafe-any
-    opn(url);
+    await vscode.env.openExternal(vscode.Uri.parse(url));
```

Comment:
"Report Issue" is a good test case, but another area I've had problems is opening the "Live Metrics stream" in the portal for Linux Function Apps. I have to do something like this:
```
`https://localhost:7071/${encodeURIComponent(JSON.stringify({ "test": "data" }))}/hi`
```

The URL looks like this:
```
https://localhost:7071/%7B%22test%22%3A%22data%22%7D/hi
```

but VS Code ends up converting it to this (which doesn't work):
```
https://localhost:7071/%7B%22test%22:%22data%22%7D/hi
```

Unfortunately I just don't think we can trust VS Code's `openExternal`. They've done a lot of work to fix edge cases, but fundamentally they're still "processing" the URL under the covers instead of just passing along the string we give them. Seems like we need to wait for https://github.com/microsoft/vscode/issues/85930 to be fixed before we can trust it